std-bits/base/src/main/java/org/enso/base/Time_Utils.java

```diff
@@ -95,6 +95,26 @@ public static String local_date_format(LocalDate date, Object format) {
     return DateTimeFormatter.ofPattern(format.toString()).format(date);
   }
 
+  public static String local_date_format(LocalDate date, Object format, Locale locale) {
+    return DateTimeFormatter.ofPattern(format.toString()).withLocale(locale).format(date);
+  }
+
+  public static String date_time_format(ZonedDateTime dateTime, Object format) {
+    return DateTimeFormatter.ofPattern(format.toString()).format(dateTime);
+  }
+
+  public static String date_time_format(ZonedDateTime dateTime, Object format, Locale locale) {
+    return DateTimeFormatter.ofPattern(format.toString()).withLocale(locale).format(dateTime);
+  }
```

Comment:
So I remember that @JaroslavTulach said that we should probably avoid using overrides in our polyglot-exported methods.

That is because having overrides may sometimes confuse the resolver (if it is unclear which override to pick) and also they have additional performance cost - apparently invoking non-overridden methods through Enso-to-Java polyglot is always slightly faster. I'm not sure how big the difference is, but I think due to these two reasons:

Ideally, we should just never use overrides in public methods that we write in our Java helper libraries that are meant to be called from Enso.

We can always just give an alternative name to each variant, making it more robust and faster.

```suggestion
  public static String date_time_format(ZonedDateTime dateTime, Object format) {
    return DateTimeFormatter.ofPattern(format.toString()).format(dateTime);
  }

  public static String date_time_format_with_locale(ZonedDateTime dateTime, Object format, Locale locale) {
    return DateTimeFormatter.ofPattern(format.toString()).withLocale(locale).format(dateTime);
  }
```